aws/table_aws_elastic_beanstalk_application_version.go

```diff
@@ -0,0 +1,248 @@
+package aws
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk/types"
+	"github.com/aws/smithy-go"
+
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk"
+
+	elasticbeanstalkv1 "github.com/aws/aws-sdk-go/service/elasticbeanstalk"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsElasticBeanstalkApplicationVersion(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_elastic_beanstalk_application_version",
+		Description: "AWS Elastic Beanstalk Application Version",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("application_name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException"}),
```

Comment:
@Priyanka-Chatterjee-2000  The `application_name` column by itself cannot serve as the KeyColumns for a get call because it does not ensure the retrieval of a single row of data. We should consider using both `application_name` and `VersionLabel` as KeyColumns for the get call. Additionally, it might be beneficial to include `application_name` as an optional qual in the list call.